ci/buildkite-pipeline.sh

```diff
@@ -305,13 +305,9 @@ EOF
     ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-flakey.sh" \
     10
 
-  command_step "local-cluster-slow-1" \
-    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-1.sh" \
-    40
-
-  command_step "local-cluster-slow-2" \
-    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-2.sh" \
-    40
+  command_step "local-cluster-slow" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow.sh" \
+    80
```

Comment:
I bumped the timeout up to 80 mins here, but the CI for local-cluster-slow completed in ~10 mins (this time). What do you think is a good value here? Back to 40?

Reply:
The passing run took ~36 mins. The two failing runs were 52 and 58 mins.